### PR TITLE
[Goonchem] Few Additional Chemicals

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Adds in two new chems for Goonchem and a new reaction

Chems

- Sarin
 - Sarin is a "scary" poison; it only does 1 tox, 1 burn, and 1 brain damage per cycle, but it torments the victim with tons of terrifying messages, makes them scream, twitch, vomit, and jitter like mad. It also has a few status effects tied with it, too: it can make you have a seizure, collapse, or become dizzy. Becomes very lethal if it remains in your system for more than 2 minutes.
 - Sarin is rapidly purged by atropine
- Atrazine
 - Atrazine is pretty much just plantbgone with a real chem name and a realistic recipe; replaces plantbgone wherever applicable

Reaction

- Chlorine Azide
 - Basically just a fixed size (small) explosion on reaction

Other
- Atropine will very very rapidly purge sarin.
- Adjusts the salt recipe to prevent some future conflicts amongst other recipes
- Adds in a "quiver" emote---used by Sarin
- Adds in a "fakevomit" proc for mobs; basically this aesthetically looks and sounds like vomiting, but it doesn't drop your nutrition or stun you.

### CODE REVIEW

**Comment by tigercat2000** [comment]
>#666
>adds chemical that makes people terrified, vomit, and damages them all over slowly

seems legit
